### ISSUE TITLE
a11y: Color Contrast - Charts

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13807]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Main Content
Foreground: #DAC342
Background: #F9F6E3
The contrast ratio is: 1.62:1
- Linux Only

Foreground: #3D98D3
Background: #FFFFFF
The contrast ratio is: 3.16:1
- Deploys on Windows
- Experimental

Foreground: #F64747
Background: #FEE4E4
The contrast ratio is: 2.94:1
- configmaps """" not found"

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 167 resolved. 
Issue 170 resolved.
